ethcore/sync/src/chain/propagator.rs

```diff
@@ -96,13 +91,45 @@ impl SyncPropagator {
 		sent
 	}
 
+	/// propagate new known pool hashes to all peers
```

Comment:
We should never propagate to all peers - this will take way too long. There are methods that select lucky peers (and are special cased for small number of peers) - we should simply use that.